device_backends/LogicalNameMapping/src/LNMMathPlugin.cc

```diff
@@ -307,6 +338,9 @@ namespace ChimeraTK { namespace LNMBackend {
       throw ChimeraTK::logic_error("This register with MathPlugin enabled is not writeable: " + _target->getName());
     }
 
+    // The readLatest might throw an exception. In this case preWrite() is never delegated and we must not call the target's portWrite().
+    _skipWriteTransfer = true;
```

Comment:
Maybe rename `_skipWriteTransfer` into `_skipWriteOperation` or `_skipWriteDelegation`? It does not only cause the transfer to be skipped but the entire write operation is not delegated to the `_target`.